simulator/simple_planning_simulator/src/simple_planning_simulator/vehicle_model/sim_model_delay_steer_acc_geared_wo_fall_guard.cpp

```diff
@@ -166,13 +166,10 @@ Eigen::VectorXd SimModelDelaySteerAccGearedWoFallGuard::calcModel(
         return pedal_acc + input(IDX_U::SLOPE_ACCX);
       }
     } else {
-      constexpr double brake_dead_band = 1e-3;
-      if (vel > brake_dead_band) {
+      if (vel > 0.0) {
         return pedal_acc + input(IDX_U::SLOPE_ACCX);
-      } else if (vel < -brake_dead_band) {
+      } else if (vel < 0.0) {
         return -pedal_acc + input(IDX_U::SLOPE_ACCX);
-      } else if (-pedal_acc >= std::abs(input(IDX_U::SLOPE_ACCX))) {
-        return 0.0;
       } else {
         return input(IDX_U::SLOPE_ACCX);
       }
```

Comment:
If we are stopped (`vel == 0`) and apply the brake (`pedal_acc < 0`) we will return `IDX_U::SLOPE_ACCX` which seems like a mistake :thinking: 
I think the case you removed `-pedal_acc >= std::abs(input(IDX_U::SLOPE_ACCX))` should be kept to return `0` when the brake is stronger than the slope acceleration.

Reply:
Exactly! Thank you

Reply:
The mentioned behavior is also achieved by the following block in update().
```
if (
    prev_state(IDX::VX) * state_(IDX::VX) <= 0.0 &&
    -state_(IDX::PEDAL_ACCX) >= std::abs(delayed_input(IDX_U::SLOPE_ACCX))) {}
```
However, I think calcModel() should maintain the physical consistency. Therefore I decide to maintain the block 
`if (-pedal_acc >= std::abs(input(IDX_U::SLOPE_ACCX)))`

Reply:
@maxime-clem 
I've fixed. Thank you so much